notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>MY_PROJECT = PROJECT_ID</p><p>MY_STAGING_BUCKET = BUCKET_URI&nbsp;# bucket should be in same region as ucaip</p><p> </p><p>Above seems unnecessary. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>